scripts/dm/checkStatus.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import argparse
+from gladier import GladierBaseClient
+from globus_sdk.exc.convert import GlobusConnectionError
+
+
+class FlowExecutionError(Exception):
+    def __init__(self, message="ERROR: Gladier flow failed."):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
I used to write exceptions like this, but I find I usually prefer now to keep exception messages simple, unless there is information that exception handlers can use to catch and react to or suppress a given exception (I would change this to simply `pass`). But that's totally up to you!